content/using/install.md

```diff
@@ -74,3 +74,101 @@ If you're running Urbit in the cloud on a small instance, you may need to additi
 Urbit wants to map 2GB of memory when it boots up. It won’t necessarily use all this memory, it just wants to see it. On a normal modern PC or Mac, or on a large cloud virtual machine, this is not an issue. On some small cloud virtual machines (Amazon or Digital Ocean) the default memory configuration is smaller than this, and you need to manually configure a swapfile.
 
 Digital Ocean has a post on adding swap [here](https://www.digitalocean.com/community/tutorials/how-to-add-swap-space-on-ubuntu-16-04). For Amazon there’s a StackOverflow thread [here](https://stackoverflow.com/questions/17173972/how-do-you-add-swap-to-an-ec2-instance).
+
+## Booting your ship
+
+Now the rubber meets the road. You'll be booting your ship with the keyfile that you [downloaded from Bridge](@/using/operations/using-bridge.md).
+
+Note that this section is only for booting a ship that uses the live Arvo network. If you just want to create an unnetworked ship for development purposes, read [this guide](@/using/develop.md#creating-a-development-ship) instead.
+
+### Step 1: Find Your Urbit ID
+
+This will look something like `~lodleb-ritrul`. You can see the name of your Urbit ID when you log into your wallet using the Bridge client.
+
+### Step 2: Find the path to your keyfile
+
+Find the absolute path to the keyfile that you downloaded from Bridge. Copy it.
+
+### Step 3: Run the boot command
+
+Type `cd` in your terminal to return to your home directory. If you want to store your ship somewhere besides your home directory, change the terminal's working directory to the desired directory.
+
+Run the command below, except with `sampel-palnet` replaced by the name of your
+Urbit identity, and `path/to/my-planet.key` replaced with the path to your keyfile:
+
+```
+./urbit -w sampel-palnet -k path/to/my-planet.key
+```
+
+Or, if you'd prefer to copy your key in, you can run:
+
+```
+./urbit -w sampel-palnet -G rAnDoMkEy
+```
+
+Either command will create a directory called `sampel-palnet/` and begin building your ship. It may take a few minutes.
+
+When your ship is finished booting, you will see the `~sampel-palnet:dojo>` prompt. At that point, you should permanently erase your keyfile from your machine.
+
+## Basic Operations
```

Comment:
Basically everything here below, IMO, should be removed, since it's duplicating content in Operations. I think since Docs owns this content, it's @rmariani's call to merge this stuff into the site.

Reply:
I think we should keep the Dojo part just so people get a taste for how to operate things. Everything below that should be cut from this doc. We can talk about making an "Admin" doc that covers such material.

Reply:
Yeah I considered that, but that page is pretty long and would be a tad overwhelming to someone new. Going through this short set of instructions should reassure a new Urbiteer that they installed everything correctly, and then they can go to the linked Operations page to learn more.